src/main/java/com/bob/jr/commands/VoiceCommands.java

```diff
@@ -152,4 +152,26 @@ public Mono<Void> tts(Intent intent) {
                 .then();
     }
 
+    public Mono<Void> saveVoice(Intent intent) {
+        final var tts = serverResources.getTextToSpeech();
+        return Mono.justOrEmpty(intent.getMessageCreateEvent())
+                .flatMap(messageCreateEvent -> {
+                    final var messageChannelOptional = messageCreateEvent.getMessage().getChannel().blockOptional();
+                    final var messageChannel = messageChannelOptional.orElseThrow();
```

Comment:
We should catch this somehow and at least log out to the server. I'm not sure how that works within Reactor.

Reply:
The error should be caught further up the react chain
https://github.com/taclayto52/bob-jr-lava/blob/fa813e5f66598bbcf6d50d400feefb2d6312d294/src/main/java/com/bob/jr/BobJr.java#L150-L159
It should get caught and printed up there.